src/cljs/rems/administration.cljs

```diff
@@ -236,9 +329,51 @@
               [[spinner/big]]
               [[:div
                 [:div.col.commands
+                 [to-create-resource-button]
                  [to-create-catalogue-item-button]]
                 [catalogue-list @catalogue @language]]])))))
 
+(defn create-resource-page []
```

Comment:
would it be neater if this was in its own ns? perhaps rems.administration.resource? I think we have too many event and subscription definitions in this ns